.utils/deploy.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# This file is used to deploy yttest to an S3 bucket. It
+# expects the S3 bucket name in an environment variable
+# $YTTEST_BUCKET
+
+set -ex
+
+if [ ! -d "test-materials" ] || [ ! -d "2019" ]; then
+    echo "Can't find test-materials/ or 2019/ directory. Are you running"\
+         "from the correct root directory?"
+    exit 1
+fi
+
+if [ -z "$YTTEST_BUCKET" ]; then
+    echo "The S3 bucket is not set. Failing."
+    exit 1
+fi
+
+# The basic strategy is to sync all the files that need special attention
+# first, and then sync everything else which will get defaults
+
+
+# For short-lived assets; in seconds
+FIVE_DAYS="432000"
+
+# For long-lived assets; in seconds
+ONE_YEAR="31536000"
+
+CSPSTATIC="\"content-security-policy\": \"default-src 'none'; "\
+"base-uri 'none'; "\
+"form-action 'none'; "\
+"object-src 'none'\""
+# CSP mimicing what https://ytlr-cert.appspot.com/2019/main.html uses
+CSP="\"content-security-policy\": \"default-src * 'unsafe-inline' 'unsafe-eval' data: blob:;\""
+HSTS="\"strict-transport-security\": \"max-age=${ONE_YEAR}; includeSubDomains; preload\""
+TYPE="\"x-content-type-options\": \"nosniff\""
+XSS="\"x-xss-protection\": \"1; mode=block\""
+REFERRER="\"referrer-policy\": \"no-referrer-when-downgrade\""
```

Comment:
Where does all this code come from? Is it used across projects? Who else from Ops/RelEng would be good to get this reviewed?

Reply:
> Where does all this code come from? Is it used across projects?

Yes, I borrowed most of it from some other projects.

> Who else from Ops/RelEng would be good to get this reviewed?

The security headers are mostly secops driven.

@g-k, can you or one of team members help review the headers in this PR and see if they're OK?

To give you some background (and not speaking on behalf of @whimboo ), the site is mainly used by developers and testers, not by public users, although anyone can see it if he/she chooses to. More details in https://bugzilla.mozilla.org/show_bug.cgi?id=1548062 and its related tickets as well.

Reply:
I'll leave this conversation open until it got a proper review from @g-k or any other team member.